PhysicsTools/PatAlgos/test/patTuple_addBTagging_cfg.py

```diff
@@ -126,31 +126,31 @@
 )
 process.patJetsAK4PF.addTagInfos = True
 
-# uncomment the following lines to add ak8PFJetsCHS with new b-tags to your PAT output
+# uncomment the following lines to add ak8PFJetsPuppi with new b-tags to your PAT output
 addJetCollection(
    process,
    labelName = 'AK8PFCHS',
-   jetSource = cms.InputTag('ak8PFJetsCHS'),
+   jetSource = cms.InputTag('ak8PFJetsPuppi'),
    jetCorrections = ('AK8PFchs', cms.vstring(['L1FastJet', 'L2Relative', 'L3Absolute']), 'Type-2'),
    algo = 'AK',
    rParam = 0.8,
    btagDiscriminators = ['pfBoostedDoubleSecondaryVertexAK8BJetTags']
 )
 process.patJetsAK8PFCHS.addTagInfos = True
 
-# uncomment the following lines to add subjets of ak8PFJetsCHSSoftDrop with new b-tags to your PAT output
+# uncomment the following lines to add subjets of ak8PFJetsPuppiSoftDrop with new b-tags to your PAT output
 from pdb import set_trace
 addJetCollection(
```

Comment:
it looks like the comment is obsolete now

Reply:
As you recommend below I will just leave the puppi out of the tests for the time being until things are integrated.